aspnetcore/blazor/hosting-models.md

```diff
@@ -78,7 +78,7 @@ There are downsides to server-side hosting:
 * Reduced scalability: The server must manage multiple client connections and handle client state.
 * An ASP.NET Core server is required to serve the app. Deployment without a server (for example, from a CDN) isn't possible.
 
-&dagger;The *blazor.server.js* script is published to the following path: *bin/{Debug|Release}/{TARGET FRAMEWORK}/publish/{APPLICATION NAME}.App/dist/_framework*.
+&dagger;The *blazor.server.js* script is served directly from an embedded resource in `.Components.Server` middleware.
```

Comment:
Microsoft.AspNetCore.Components.Server is now part of the ASP.NET Core shared framework, so users won't typically see it.  Maybe say "from an embedded resource in the ASP.NET Core shared framework"?